src/js/tabs/vtx.js

```diff
@@ -608,15 +608,8 @@ TABS.vtx.initialize = function (callback) {
     }
 
     function save_lua() {
-        const suggestedName = 'model01';
         const suffix = 'lua';
-
-        let filename;
-        if(FC.CONFIG.name && FC.CONFIG.name.trim() !== '') {
-            filename = FC.CONFIG.name.trim().replace(' ', '_');
-        }else{
-            filename = suggestedName;
-        }
+        let filename = FC.CONFIG.uid[0].toString(16).padStart(8, '0') + FC.CONFIG.uid[1].toString(16).padStart(8, '0') + FC.CONFIG.uid[2].toString(16).padStart(8, '0');
         filename += `.${suffix}`;
```

Comment:
I think is better to join the `suffix` now in the name, now that the generation of the name it's unique:
```javascript
const filename = `${FC.CONFIG.uid[0].toString(16).padStart(8, '0')}${FC.CONFIG.uid[1].toString(16).padStart(8, '0')}${FC.CONFIG.uid[2].toString(16).padStart(8, '0')}.${suffix};
```

Or more clear:
```javascript
const uid0 = FC.CONFIG.uid[0].toString(16).padStart(8, '0');
const uid1 = FC.CONFIG.uid[1].toString(16).padStart(8, '0');
const uid2 = FC.CONFIG.uid[2].toString(16).padStart(8, '0');

const filename = `${uid0}${uid1}${uid2}.${suffix}`;
```

Reply:
DOne 👍